solutions/api/src/lambda/token/utils/getAuthRequest.ts

```diff
@@ -0,0 +1,64 @@
+import { getDynamoDbClient } from "../../../../../commons/utils/awsClient/dynamodbClient/index.js";
+import { errorManager } from "./errors.js";
+import * as v from "valibot";
+
+export const getAuthRequest = async (
+  code: string,
+  assertion_redirect_uri: string | undefined,
```

Comment:
`redirect_uri` is meant to come in the incoming request as a 1st class attribute, not in the client assertion jwt.